src/components/Achievement/Achievement.tsx

```diff
@@ -3,8 +3,8 @@ import Image from "next/image";
 
 const Achievement = () => {
   return (
-    <div className="relative max-w-[1440px] mx-auto">
-      <div className="relative z-[11] py-16 grid gap-8 sm:px-0 md:px-16 md:py-18 lg:grid-cols-2 lg:gap-0">
+    <div className="bg-gradient-to-b from-[#F1EDFD] to-white">
+      <div className="max-w-[1440px] mx-auto py-16 grid gap-8 sm:px-0 md:px-16 md:py-18 lg:grid-cols-2 lg:gap-0">
```

Comment:
@sweta1308, let's create a class and put all of this repetitive styling in there.

```css
.custom-container{
@apply max-w-[1440px] mx-auto;
}
```

Reply:
Updated